x-pack/metricbeat/module/meraki/meraki.go

```diff
@@ -0,0 +1,164 @@
+package meraki
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/elastic/beats/v7/metricbeat/mb"
+	"github.com/elastic/elastic-agent-libs/mapstr"
+
+	meraki_api "github.com/meraki/dashboard-api-go/v3/sdk"
+)
+
+// ModuleName is the name of this module.
+const ModuleName = "meraki"
+
+// func init() {
```

Comment:
Yes, I removed all unused code. I had some debug code still in there, among other things ... although the above was actually used, but I do not believe it will be used in refactor.  This meraki.go will likely be gone in refactor, the intent was to avoid code duplication to put any function, common across metricsets in here, since we are consolidating to a single device_health, this will be no longer necessary.